app/config/parameters.yml.dist

```diff
@@ -5,23 +5,12 @@
 parameters:
     # this demo application uses an embedded SQLite database to simplify setup.
     # in a real Symfony application you probably will use a MySQL or PostgreSQL database
-    database_driver:   pdo_sqlite
-    database_host:     127.0.0.1
-    database_port:     ~
-    database_name:     ~
-    database_user:     root
-    database_password: ~
-    # the 'database_path' is only used for SQLite type databases
-    database_path:     '%kernel.root_dir%/data/blog.sqlite'
+    # the path must be relative or else it will not work on Windows
+    database_url:      'sqlite:///%kernel.root_dir%/data/blog.sqlite'
 
     # Uncomment these lines to use a MySQL database instead of SQLite:
     #
-    # database_driver: pdo_mysql
-    # database_host: 127.0.0.1
-    # database_port: ~
-    # database_name: symfony_demo
-    # database_user: root
-    # database_password: ~
+    # database_url: mysql://root@pass127.0.0.1:3306/symfony_demo
```

Comment:
Ah yes @bocharsky-bw thanks.

Using URLs for this purpose is not proprietary to Heroku, by the way ;) Doctrine supports them (that's why they work here), and many other languages/frameworks (Rails, Django, ...) because it make management of credentials much less of a hassle especially if you do it through the environment in e.g. Docker or 12-factor apps - `redis://..`, `smtp://user:pass@server:1234/` and so forth, they work for anything.